src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -756,6 +756,30 @@ static double IFloatingPoint<double>.Tanh(double x)
         static double IFloatingPoint<double>.Truncate(double x)
             => Math.Truncate(x);
 
+        [RequiresPreviewFeatures]
+        static bool IFloatingPoint<double>.IsFinite(double d) => IsFinite(d);
```

Comment:
I didn't add tests for the new APIs since I didn't find tests for the other generic math APIs in Double/Single/Half. I assumed we'll add them as part of adding other unit tests here.